scrapper.py

```diff
@@ -21,43 +32,189 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+class HTMLParser:
+    """
+    Parser implementation
+    """
+    def __init__(self, article_url, article_id):
+        """
+        Init
+        """
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(url=article_url, article_id=article_id)
+
+    def parse(self):
+        """
+
+        """
+        response = requests.get(self.article_url)
+        article_bs = BeautifulSoup(response.text, 'html.parser')
+
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        self.article.save_raw()
+
+        return self.article
+
+    def _fill_article_with_text(self, article_bs):
+        """
+        Fills self.article with text from article_bs
+        """
+
+        table_rows = article_bs.find_all('tr', {"class": "unnrow"})
+
+        for table_row in table_rows:
```

Comment:
do we need a loop to find the only button?

Reply:
Yes, the amount of table_rows may vary